base/linalg/blas.jl

```diff
@@ -112,6 +112,29 @@ function set_num_threads(n::Integer)
     return nothing
 end
 
+"""
+    get_num_threads(n)
+
+Get the number of threads the BLAS library can use.
+"""
+function get_num_threads()
```

Comment:
Since I've realized the behavior of get_num_threads is more complicated than I thought I'm not going to add it BLAS in this pull request. I stayed with the try in the test.